cli/src/api/templates/load-wasi-template.ts

```diff
@@ -144,11 +144,7 @@ const __wasmDebugFilePath = __nodePath.join(__dirname, '${wasmFileName}.debug.wa
 if (__nodeFs.existsSync(__wasmDebugFilePath)) {
   __wasmFilePath = __wasmDebugFilePath
 } else if (!__nodeFs.existsSync(__wasmFilePath)) {
-  try {
-    __wasmFilePath = __nodePath.resolve('${packageName}-wasm32-wasi')
-  } catch {
-    throw new Error('Cannot find ${wasmFileName}.wasm file, and ${packageName}-wasm32-wasi package is not installed.')
-  }
+  __wasmFilePath = __nodePath.resolve('node_modules/${packageName}-wasm32-wasi/${wasmFileName}.wasm')
```

Comment:
```suggestion
  __wasmFilePath = __nodePath.resolve('${packageName}-wasm32-wasi/${wasmFileName}.wasm')
```

In some installation modes of package managers, `${packageName}-wasm32-wasi` may not be under the `node_modules` of the same directory (hoist)